### ISSUE TITLE
[🐛 BUG] Applying width to a specific column in a tgb.table does not work

### ISSUE THREAD

**Issue by AlexandreSajus** [Avaiga/taipy#2286]
### What went wrong? 🤔

A customer requested for specific columns in his tgb.table to have a bigger width. The column names have spaces in them so using tgb.table(width__column_name="200px") won't work.

This example is supposed to work but applies the style to all columns instead:
```python
from taipy.gui import Gui
import taipy.gui.builder as tgb
import pandas as pd

data = pd.DataFrame({"A C": [1, 2, 3], "B C": [4, 5, 6], "C C": [7, 8, 9]})

properties_table = {"width[B C]": "500px"}

with tgb.Page() as page:
    tgb.table(data="{data}", properties="{properties_table}")

Gui(page=page).run()
```
![image](https://github.com/user-attachments/assets/5636166c-59be-4c08-92f6-a1427cad5cd1)

### Acceptance Criteria

- [ ] A unit test reproducing the bug is added.
- [ ] Any new code is covered by a unit tested.
- [ ] Check code coverage is at least 90%.
- [ ] The bug reporter validated the fix.
- [ ] Related issue(s) in taipy-doc are created for documentation and Release Notes are updated.

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [ ] I am willing to work on this issue (optional)

**Comment by FredLL-Avaiga** [comment]
This issue is not linked to the column name.
the width for the column is propagated to the front-end but it was not noticed due to the way table handle column size requests.

**Comment by FredLL-Avaiga** [comment]
A css workaround

```py
with tgb.Page(style={"taipy-table": {"th:nth-child(2)": {"min-width": "500px"}}}) as page:
    tgb.table(data="{data}", properties="{properties_table}")

```

**Comment by FlorianJacta** [comment]
I believe this is not fixed. I cannot change the width of a single column

**Comment by FredLL-Avaiga** [comment]
Is it fixed in develop ?

**Comment by FlorianJacta** [comment]
Same thing. It seems like changing the width of one column is changing everything.

If I configure every width for every column, I have a different behavior:

```python
from taipy.gui import Gui
import taipy.gui.builder as tgb
import pandas as pd

data = pd.DataFrame({"A C": [1, 2, 3], "B C": [4, 5, 6], "C C": [7, 8, 9]})

properties_table = {"width[B C]": "50px", "width[A C]": "500px", "width[C C]": "100px"}

with tgb.Page() as page:
    tgb.table(data="{data}", properties="{properties_table}")

Gui(page=page).run()

```

**Comment by AlexandreSajus** [comment]
Fixed in 4.0.2.dev2
